src/components/Checkout/DonationItem.vue

```diff
@@ -1,184 +1,13 @@
-								data-testid="basket-donation-edit-button-mobile-amount"
-								v-kv-track-event="['basket', 'Edit Donation']"
-								@click="enterEditDonation"
-								title="Edit Donation"
-							>
-								{{ formattedAmount }}
-								<kv-material-icon
-									role="img"
-									aria-label="Edit Donation"
-									title="Edit Donation"
-									class="edit-donation tw-text-action
-									tw-w-3.5
-									tw-h-3.5
-									tw-py-0.5
-									tw-hidden md:tw-inline-block
-									"
-									name="pencil"
-									:icon="mdiPencil"
-								/>
-							</button>
-						</div>
-
-						<div>
-							<div
-								class="donation-tagline tw-text-small tw-text-secondary tw-my-1"
-								data-testid="basket-donation-tagline"
-								v-html="donationTagLine"
-							>
-							</div>
-							<button
-								class="tw-text-small tw-text-link"
-								data-testid="basket-donation-info-lightbox"
-								@click="triggerDefaultLightbox"
-								v-kv-track-event="['basket', 'Donation Info Lightbox', 'Open Lightbox']"
-							>
-								{{ donationDetailsLink }}
-							</button>
-						</div>
-					</div>
-				</div>
-
-				<!-- donation total -->
-				<div
-					class="
-				tw-flex-none
-				tw-w-full
-				md:tw-w-auto
-				md:tw-ml-3
-				lg:tw-ml-4.5
-				tw-mt-1.5
-				md:tw-mt-0"
-				>
-					<div
-						v-show="!editDonation"
-						class="tw-hidden md:tw-block tw-text-right"
-					>
-						<button
-							class="donation-amount"
-							data-testid="basket-donation-edit-button-combined"
-							v-kv-track-event="['basket', 'Edit Donation']"
-							@click="enterEditDonation"
-							title="Edit Donation"
-						>
-							{{ formattedAmount }}
-							<kv-material-icon
-								role="img"
-								aria-label="Edit Donation"
-								title="Edit Donation"
-								class="edit-donation tw-text-action
-								tw-w-3.5
-								tw-h-3.5
-								tw-py-0.5
-								tw-align-bottom
-								"
-								name="pencil"
-								:icon="mdiPencil"
-							/>
-						</button>
-					</div>
-					<div v-show="editDonation" class="small-12 columns donation-amount-input-wrapper">
-						<kv-text-input
-							class="donation-amount-input"
-							data-testid="basket-donation-edit-input"
-							name="donation"
-							id="donation"
-							v-model="amount"
-							@blur="validateInput"
-							@keyup.enter.prevent="updateDonation()"
-						/>
-						<kv-button
-							variant="secondary"
-							class="update-donation-inline-button"
-							data-testid="basket-donation-edit-submit"
-							@click="updateDonation()"
-						>
-							Update
-						</kv-button>
-						<button
-							class="show-for-medium remove-wrapper"
-							@click="updateLoanAmount('remove')"
-							data-testid="basket-donation-remove"
-						>
-							<kv-material-icon
-								class="remove-x tw-text-tertiary"
-								name="small-x"
-								:from-sprite="true"
-								title="Remove donation"
-							/>
-						</button>
-					</div>
-					<donate-repayments
```

Comment:
Why is this being completely removed? What other pathway do we expect lenders to use to activate this condition on their loans?

Reply:
Ok, I do see this in one of the code blocks below so looks like it will still remain. Sorry for the confusion.